test/datomish/test/query.cljc

```diff
@@ -824,3 +824,72 @@
                             '[?save :save/excerpt ?excerpt]]))]
       (is (or (= ["Some page title" "Some page excerpt"] result)
               (= ["A different page" "A different excerpt"] result))))))
+
+(deftest-db test-or-join-real-world conn
```

Comment:
Consider a name that shows what you're testing, and perhaps a `testing` block to describe it.